pkg/noderesourcetopology/cache/overreserve_test.go

```diff
@@ -652,35 +657,38 @@ func dumpNRT(nrtObj *topologyv1alpha2.NodeResourceTopology) string {
 	return string(nrtJson)
 }
 
-type fakePodByNodeNameIndex struct {
+type fakePodLister struct {
 	pods []*corev1.Pod
 	err  error
 }
 
-func (fi *fakePodByNodeNameIndex) Add(pod *corev1.Pod) {
-	fi.pods = append(fi.pods, pod)
+type fakePodNamespaceLister struct {
+	parent    *fakePodLister
+	namespace string
 }
 
-func (fi *fakePodByNodeNameIndex) GetPodNamespacedNamesByNode(logID, nodeName string) ([]types.NamespacedName, error) {
-	if fi.err != nil {
-		return nil, fi.err
-	}
+func (fpl *fakePodLister) AddPod(pod *corev1.Pod) {
+	fpl.pods = append(fpl.pods, pod)
+}
 
-	var objs []types.NamespacedName
-	for _, pod := range fi.pods {
-		if pod.Spec.NodeName != nodeName {
-			continue
-		}
-		objs = append(objs, types.NamespacedName{
-			Namespace: pod.GetNamespace(),
-			Name:      pod.GetName(),
-		})
+func (fpl *fakePodLister) List(selector labels.Selector) ([]*corev1.Pod, error) {
+	return fpl.pods, fpl.err
+}
+
+func (fpl *fakePodLister) Pods(namespace string) podlisterv1.PodNamespaceLister {
+	return &fakePodNamespaceLister{
+		parent:    fpl,
+		namespace: namespace,
 	}
-	return objs, nil
 }
 
-func (fi *fakePodByNodeNameIndex) TrackReservedPod(pod *corev1.Pod, nodeName string)   {}
-func (fi *fakePodByNodeNameIndex) UntrackReservedPod(pod *corev1.Pod, nodeName string) {}
+func (fpnl *fakePodNamespaceLister) List(selector labels.Selector) ([]*corev1.Pod, error) {
```

Comment:
Are this going to be implement at some point or they're just needed for the interface implementation?